src/main/java/org/broadinstitute/hellbender/tools/copynumber/GermlineCNVCaller.java

```diff
@@ -0,0 +1,408 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import org.broadinstitute.barclay.argparser.Advanced;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.ArgumentCollection;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
```

Comment:
Is this tool out of Beta status now?

Reply:
This is a complete python-based rewrite of the old Spark-based GermlineCNVCaller, so it should still be marked Beta.

Reply:
Done.